src/DevTiqrClientBundle/Command/AuthenticationCommand.php

```diff
@@ -28,6 +28,7 @@
 
 require_once __DIR__.'/../../../vendor/tiqr/tiqr-server-libphp/library/tiqr/Tiqr/OATH/OCRA.php';
 
+/** @SuppressWarnings(PHPMD) */
```

Comment:
Seeing this will not be merged back to develop I wont require changes to this warning suppression. But usually we suppress only the conflicting sniff. Feel free to update or leave as be.